app/javascript/plugins/fundraiser/FundraiserView.js

```diff
@@ -98,6 +113,15 @@ export class FundraiserView extends Component {
       </div>
     ) : null;
 
+    const supportedCurrencyDisclaimer = !supportedLocalCurrency ? (
+      <div className="currency-disclaimer fundraiser-bar">
+        <FormattedMessage
+          id="fundraiser.unsupported_currency.one_click_failed"
```

Comment:
@Tuuleh isn't this the wrong ID? Also, for `defaultMessage` I think we're just passing the EN string value - it's to cover our backs when the translation is missing.

Reply:
Yes, it's the wrong ID. Thanks. I'll clean this up (I sloppily copied it from above). I have a clean up commit coming up where I've also removed the lingering console.log. I'll look at the api of FormattedMessage and how to pass it the translations as well as a default msg. I'm really drowning in this PT work right now so if you have capacity to help out on it tomorrow, that'd be awesome (I haven't even gotten the QA report for Champaign yet and I'll already have work for tomorrow).